tests/helpers.js

```diff
@@ -17,7 +17,7 @@ export const fakeMessageData = {
 export const EMPTY_PNG = Buffer
   .from(oneLine`iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAACklEQVR4nGMA
                 AQAABQABDQottAAAAABJRU5ErkJggg==`,
-        'base64')
```

Comment:
uhm, why? This is an argument to `from` so it should be indented like that? Oh, because two whitespaces? Maybe an idea to indent the base64 string too, mhmm, I don't care enough I guess